conda/exceptions.py

```diff
@@ -731,6 +733,15 @@ def __init__(self, bad_deps, chains=True, strict=False):
         super(UnsatisfiableError, self).__init__(msg)
 
 
+class RawStrUnsatisfiableError(UnsatisfiableError):
```

Comment:
We could move this to `conda-libmamba-solver` if needed.

Reply:
Interesting, I think this is an implementation detail of how CLS interacts with libmamba? If yes, let's move it there.